File: javascript/Baekjoon/Bronze/I/1546.js
// 평균
// https://www.acmicpc.net/problem/1546

let input = require('fs').readFileSync('../../예제.txt').toString().replace(/\r*/g, "").trim().split('\n');
let count = Number(input.shift());
input = input[0].split(' ').map((num) => Number(num));
let result = 0;
let max = Math.max.apply(null, input);

for(let i = 0; i < count; i++) {
    result += input[i] / max * 100;
}

console.log(result / count);

File: Python/beakjoon/Bronze/I/1546.py
# 평균
# https://www.acmicpc.net/problem/1546

count = int(input())
num = list(map(int, input().split()))
maxNum = max(num)
average = 0.0

for i in num:
    average += i / maxNum * 100

print(average / count)